django_stripe/models/abstracts/core/customers.py

```diff
@@ -84,7 +94,7 @@ class AbstractStripeCustomer(AbstractStripeModel):
         help_text="Describes the customer's tax exemption status. When set to reverse, "
         'invoice and receipt PDFs include the text "Reverse charge"',
     )
-    preferred_locales = ArrayField(
+    preferred_locales = LocalesField(
         models.CharField(default="", blank=True, max_length=255),
```

Comment:
@mjtorn We can create abstract model class that could that will have postgres arrayfield and custom arrayfield based on database engine.